scheds/rust/scx_layered/src/main.rs

```diff
@@ -1240,6 +1240,21 @@ impl<'a> Scheduler<'a> {
     }
 
     fn init_cpu_prox_map(topo: &Topology, cpu_ctxs: &mut Vec<bpf_intf::cpu_ctx>) {
+        let radiate = |mut vec: Vec<usize>, center_id: usize| -> Vec<usize> {
+            vec.sort_by_key(|&id| ((id as i32 - center_id as i32).abs(), id.cmp(&center_id)));
+            vec
+        };
+        let radiate_cpu =
+            |mut vec: Vec<usize>, center_cpu: usize, center_core: usize| -> Vec<usize> {
+                vec.sort_by_key(|&id| {
+                    (
+                        (topo.all_cpus.get(&id).unwrap().core_id as i32 - center_core as i32).abs(),
+                        center_cpu.cmp(&id),
```

Comment:
Would it be better to also use abs distance from center_cpu here? Both sibling CPU IDs could be on the same side of the center and in such cases `center_cpu.cmp(&id)` wouldn't order them in a deterministic way. This likely doesn't matter in any practical sense but it does make the code confusing as it's unclear what the second ordering is supposed to do. The same for goes for `radiate`.

Reply:
That makes sense, it was completely backwards. Should be good now.